templates/olm-catalog/hyperconverged-operator/VERSION/hyperconverged.VERSION.csv.yaml.in

```diff
@@ -0,0 +1,110 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kubevirt-hyperconverged.{{.CsvVersion}}
+  namespace: placeholder
```

Comment:
kubevirt?

Reply:
I don't know if `kubevirt-hyperconverged` is the correct answer but we do need a name more descriptive than `hyperconverged-cluster-operator`. That name is fine on github and as a go project as it is under kubevirt.

Reply:
I use "placeholder" as namespace in the KubeVirt CSV too, because I saw it in several other CSVs, and it works nice at least with OLM + Marketplace. Seems to be replaced by one of them.